make/test/BuildTestSetupAOT.gmk

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+include MakeFileStart.gmk
+
+################################################################################
+# This file builds the TestSetupAOT.class, which is used by SetupAOT
+# in ../RunTests.gmk
+################################################################################
+
+include CopyFiles.gmk
+include JavaCompilation.gmk
+
+################################################################################
+
+SETUP_AOT_BASEDIR := $(TOPDIR)/test/setup_aot
+SETUP_AOT_SUPPORT := $(SUPPORT_OUTPUTDIR)/test/setup_aot
+SETUP_AOT_CLASS := $(SETUP_AOT_SUPPORT)/classes/TestSetupAOT.class
+
+$(eval $(call SetupJavaCompilation, BUILD_SETUP_AOT, \
+    TARGET_RELEASE := $(TARGET_RELEASE_NEWJDK_UPGRADED), \
+    SRC := $(SETUP_AOT_BASEDIR), \
+    BIN := $(SETUP_AOT_SUPPORT)/classes, \
+))
+
+TARGETS += $(BUILD_SETUP_AOT)
+
+################################################################################
+# Targets for building test-image.
+################################################################################
+
+# Copy to hotspot jtreg test image
+$(eval $(call SetupCopyFiles, COPY_SETUP_AOT, \
+    SRC := $(SETUP_AOT_SUPPORT)/classes, \
+    DEST := $(TEST_IMAGE_DIR)/setup_aot, \
+    FILES := TestSetupAOT.class, \
+))
+
+IMAGES_TARGETS += $(COPY_SETUP_AOT)
+
+build: $(TARGETS)
```

Comment:
The target `build` is no longer the default target in this makefile, so this line is dead code unless you change the call in Main.gmk by setting `TARGET := build`. You can also just remove this line as the default target `all` gets setup to depend on `$(TARGETS)` anyway in MakeFileEnd.gmk.

Reply:
I removed this line and verified TestSetupAOT.class is still created inside the test image.